Vorlesungsbeispiele/MRT2_VL-4_CalcAsFSM/example.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <iostream>
 #include <fstream>
+#include <list>
```

Comment:
Please check this again: Where do we need it?

Reply:
We don't, actually. List is already imported in Automaton header. The project wouldn't compile at first so this line was part of troubleshooting. But I think the problem was a wrongly defined include path, and this line ended up here sort of by accident.